src/chapters/introduction-and-setup/setup.rst

```diff
@@ -111,6 +113,66 @@ the capability to compile single-file Java programs without explicitly running a
 ``Hello, World`` program were more complex and contained another file, we would need to first run 
 ``javac HelloWorld.java``, to compile, followed by ``java HelloWorld.java``.
 
+Windows Users
+^^^^^^^^^^^^^
+
+Let's write a simple "Hello, World" program and watch the JDK in action. 
+
+In the future, we'll be doing most of our Java coding with the IntelliJ IDE. 
+IntelliJ contains many features to help us write Java properly and easily, 
+including its own compiler. For now though, we'll use a simpler text editor 
+so we can demonstrate what we get with the JDK.
+
+In the text editor of your choice, create and save a file called 
+``HelloWorld.java`` and include the code below:
+
+.. sourcecode:: java
+   :linenos:
+
+   public class HelloWorld {
+
+      public static void main(String[] args) {
+
+         System.out.println("Hello, World");
+      }
+
+   }
+
+We'll discuss the syntax of this program soon, but you can likely trust your gut
+that this program has an expected output of "Hello, World". 
+
+To test this hypothesis, open a terminal window and navigate to the parent directory of your new file.
+In a separate window, navigate to the Java Development Kit to get the file path. Copy the file path.
```

Comment:
As an alternative, Windows users can use the file explorer, which helps those who prefer a GUI.  How do you feel about using that method instead of the terminal?

Either way, can we add a screenshot or a file tree to help the students know where to look for the JDK folder?  There were two "Program Files" folders on the LC machine (one with (x86) added to the name), so more details for getting the file path would be helpful.

Reply:
Done! Let me know your thoughts!